README.md

```diff
@@ -10,15 +10,15 @@
    - [Patch Release Manager](#patch-release-manager)
    - [Kubernetes Release Management Team for Major/Minor Releases](#kubernetes-release-management-team-for-majorminor-releases)
    - [Release Team Lead](#release-team-lead)
-   - [Release Team Secondary](#release-team-secondary)
+   - [Release Team Shadow](#release-team-shadow)
    - [Release Branch Manager](#release-branch-manager)
    - [Docs Lead](#docs-lead)
    - [Features Lead](#features-lead)
    - [Bug Triage Lead](#bug-triage-lead)
    - [Test Infra Lead](#test-infra-lead)
    - [Upgrade Testing Lead](#upgrade-testing-lead)
    - [CI Signal Lead](#ci-signal-lead)
-   - [Release Team Shadow](#release-team-shadow)
+   - [Release Team Lead Shadow](#release-team-lead-shadow)
```

Comment:
This TOC entry should be swapped with the one you changed above.

Reply:
@enisoc oooops, fixed.
Thanks!